### PR TITLE
Add build script for i.MX8, i.MX8X and i.MX8M

### PR DESCRIPTION
**Pull request by dianacretu**
Signed-off-by: Diana Cretu <diana.cretu@nxp.com>

### CODE REVIEW

File: build-sof.sh [hunk 1]
```diff
@@ -24,6 +26,10 @@ cp config-apl-gcc8.1-gdb8.1 .config
 ./ct-ng build
 cp config-cnl-gcc8.1-gdb8.1 .config
 ./ct-ng build
+cp config-imx-gcc8.1-gdb8.1 .config
```
Comment (lgirdwood) [inline build-sof.sh]: We are using gcc 9.x now.

**Review by lgirdwood — COMMENTED** [review]
Can you check the CC version - everyone should be using 9.x now and if so the whole script needs updated.

File: build-sof.sh (hunk 1 above)
Comment (dbaluta) [reply build-sof.sh]: @dianacretu, indeed we switched to  this branch of crosstool-ng:

https://github.com/thesofproject/crosstool-ng/tree/sof-gcc9x

So, these are the new config files for imx:
* imx8/imx8x -  config-imx-gcc9.2-gdb8.3 
* imx8m -  config-imx8m-gcc9.2-gdb8.3

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@dianacretu  I guess we need to update the GCC version for all the targets in this script. i.e. It's probably worth having a script variable for GCC/binutils versions. Care to do this as a subsequent PR ?

**Pull request merged**